EnglishTranslation-master/QUEST.tsv

```diff
@@ -5449,7 +5449,7 @@ QUEST_20170303_005445	Enter monster level.{nl}(Max. value: 1 ~ 500)	몬스터를
 QUEST_20170303_005446	Enter monster MHP.{nl}(Max. value: 1 ~ 500)	몬스터를 MHP을 입력하세요.{nl}(입력 가능한 값 : 1 ~ 100)				몬스터 소환
 QUEST_20170313_005448	Historic Site Ruins (Recommended Lv : 80){nl}Only you and your party members can enter together.{nl}Are you sure you want to enter?	폐허 유적지(권장레벨:80)으로, 당신과 파티를 맺은 플레이어만이 함께 입장할 수 있습니다.{nl}입장하시겠습니까?	QUEST_20150918_002406	Historic Site Ruins (Recommended Lv : 90){nl}Only you and your party members can enter together.{nl}Are you sure you want to enter?	폐허 유적지(권장레벨:90)으로, 당신과 파티를 맺은 플레이어만이 함께 입장할 수 있습니다.{nl}입장하시겠습니까?	폐허 유적지
 QUEST_20170313_005449	Demon Prison Guild (Recommended Lv : 110){nl}Only you and your party members can enter together.{nl}Are you sure you want to enter?	염원의 비석로(권장레벨:110)으로, 당신과 파티를 맺은 플레이어만이 함께 입장할 수 있습니다.{nl}입장하시겠습니까?	QUEST_20151224_003030	Demon Prison Guild (Recommended Lv : 115){nl}Only you and your party members can enter together.{nl}Are you sure you want to enter?	염원의 비석로(권장레벨:115)으로, 당신과 파티를 맺은 플레이어만이 함께 입장할 수 있습니다.{nl}입장하시겠습니까?	염원의 비석로
-QUEST_20170313_005450	Hello, I'm Identifier Sandra.{nl}Anything I can help you with?	안녕하세요. 저는 감정사 산드라입니다.{nl}제게 볼 일이 있으신가요?	QUEST_20161214_005002	Finding things that are hidden is always an interesting job.{nl}I'm sure there's something in it for you too.{nl}Alright, let me have a look.	숨겨진 걸 알아내는 건 언제나 즐거운 작업이에요.{nl}분명 당신께도 도움이 될 거에요.{nl}자! 제게 맡겨 보세요.	감정사 산드라
+QUEST_20170313_005450	Hello, I'm Sandra.{nl}Anything I can help you with?	안녕하세요. 저는 감정사 산드라입니다.{nl}제게 볼 일이 있으신가요?	QUEST_20161214_005002	Finding things that are hidden is always an interesting job.{nl}I'm sure there's something in it for you too.{nl}Alright, let me have a look.	숨겨진 걸 알아내는 건 언제나 즐거운 작업이에요.{nl}분명 당신께도 도움이 될 거에요.{nl}자! 제게 맡겨 보세요.	감정사 산드라

```

Comment:
the original translation included her title. There are many Appraisers/Identifiers in the world, she's just specifying her title.

Reply:
Debatable. 

In my opinion, for a minor NPC, it is unnecessary to keep using the title as her title and name can already be seen above her dialogue. She's also the only person with this title as an NPC. There's also a line in ETC that says `Unidentified items can be identified at the Appraiser Sandra NPC`.

Reply:
She's not a minor NPC, she's the job master for Appraiser which is a very important NPC. If anything, the translation of "Identifier" in all cases should be verified if it should be "Appraiser."